server/src/db/seeders/ReviewsSeeder.js

```diff
@@ -0,0 +1,47 @@
+import { Review } from "../../models/index.js"
+
+class ReviewsSeeder {
+    static async seed() {
+        const reviewsData = [
+            {
+                review: "Great Bar and Grille fun Irish vibes!",
+                userId: 1,
+                activityId: 1
```

Comment:
we cannot hard code foreign keys in the seeder - consult your other seeders for pattern on querying this data and using it's `id` property directly. We cannot assume there will be a record with that corresponding id